python/ray/train/v2/_internal/constants.py

```diff
@@ -13,12 +13,12 @@
 CHECKPOINT_MANAGER_SNAPSHOT_FILENAME = "checkpoint_manager_snapshot.json"
 
 
-# ------------------------------------------------------------
-# Environment variables used in the controller and workers.
+# -----------------------------------------------------------------------
+# Environment variables used in the controller, workers, and state actor.
 #
 # Be sure to update ENV_VARS_TO_PROPAGATE when adding new
 # environment variables in this section.
-# ------------------------------------------------------------
+# -----------------------------------------------------------------------
 
 # Polling interval for the Train controller.
 # This determines how many seconds the controller will wait between
```

Comment:
Not critical but I do think just passing these in as part of the constructor would be cleaner and more explicit.